presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/ResourceGroupsDao.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.resourceGroups.db;
+import org.skife.jdbi.v2.sqlobject.Bind;
+import org.skife.jdbi.v2.sqlobject.SqlQuery;
+import org.skife.jdbi.v2.sqlobject.SqlUpdate;
+import org.skife.jdbi.v2.sqlobject.customizers.Mapper;
+
+import java.util.List;
+
+public interface ResourceGroupsDao
+{
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS resource_groups_global_properties (\n" +
+            "cpu_quota_period VARCHAR(128) NULL)")
+    void createResourceGroupsGlobalPropertiesTable();
+
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS resource_groups (\n" +
+            "resource_group VARCHAR(250) NOT NULL,\n" +
+            "soft_memory_limit VARCHAR(128) NOT NULL,\n" +
+            "max_queued INT NOT NULL,\n" +
+            "max_running INT NOT NULL,\n" +
+            "scheduling_policy VARCHAR(128) NULL,\n" +
+            "scheduling_weight INT NULL,\n" +
+            "jmx_export BOOLEAN NULL,\n" +
+            "soft_cpu_limit VARCHAR(128) NULL,\n" +
+            "hard_cpu_limit VARCHAR(128) NULL,\n" +
+            "PRIMARY KEY (resource_group)" +
+            ")")
+    void createResourceGroupsTable();
+
+    @SqlQuery("SELECT cpu_quota_period FROM resource_groups_global_properties;")
+    @Mapper(ResourceGroupGlobalProperties.Mapper.class)
+    List<ResourceGroupGlobalProperties> getResourceGroupGlobalProperties();
+
+    @SqlUpdate("INSERT INTO resource_groups_global_properties\n" +
+            "(cpu_quota_period) VALUES (:cpu_quota_period)")
+    void insertResourceGroupsGlobalProperties(
+            @Bind("cpu_quota_period") String cpuQuotaPeriod
+    );
+
+    @SqlUpdate("UPDATE resource_groups_global_properties set cpu_quota_period = :cpu_quota_period")
+    public void updateResourceGroupsGlobalProperties(
```

Comment:
That's for the future (i.e. alter the resource groups via sql commands instead of updating the db). I could remove it and add it back when/if we're ready to do that.